public/src/components/channelManagement/campaigns/CampaignsList.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import { List, ListItem, Theme, makeStyles, Button, Typography } from '@material-ui/core';
+import { Campaigns, Campaign } from './CampaignsForm';
+
+const useStyles = makeStyles(({}: Theme) => ({
+  container: {
+    marginTop: '16px',
+  },
+  list: {
+    padding: 0,
+    width: '100%',
+  },
+  listItem: {
+    margin: 0,
+    padding: 0,
+    gutter: 0,
+    width: '100%',
+  },
+  button: {
+    position: 'relative',
+    height: '50px',
+    width: '290px',
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'space-between',
+    background: 'white',
+    borderRadius: '4px',
+    padding: '0 12px',
+    marginBottom: '4px',
+  },
+  text: {
+    fontSize: '12px',
+    fontWeight: 500,
+    textTransform: 'uppercase',
+    letterSpacing: '1px',
+  },
+}));
+
+interface CampaignsListProps {
+  campaigns: Campaigns;
+  selectedCampaign?: Campaign;
+  onCampaignSelected: (testName: string) => void;
+}
+
+const CampaignsList = ({
+  campaigns,
+  selectedCampaign,
+  onCampaignSelected,
+}: CampaignsListProps): React.ReactElement => {
+  const classes = useStyles();
+
+  const checkIfCampaignIsSelected = (): 'outlined' => {
```

Comment:
TODO (but lower priority): add in the visual signalling surrounding whether a campaign has been selected for viewing (and eventually, editing)